client/reader/sidebar/index.jsx

```diff
@@ -49,6 +52,10 @@ const ReaderSidebar = React.createClass( {
 
 		// If we're browsing a tag or list, open the sidebar menu
 		this.openExpandableMenuForCurrentTagOrList();
+
+		if ( isEmpty( this.props.subscribedLists ) ) {
```

Comment:
Need some extra logic here to stop us from kicking off the fetch repeatedly when the user has no lists.